--- snapper-0.8.15/configure.ac~	2021-03-15 21:29:58.000000000 +0100
+++ snapper-0.8.15/configure.ac	2021-03-15 21:33:42.630871897 +0100
@@ -194,9 +194,7 @@
 
 PKG_CHECK_MODULES(DBUS, dbus-1)
 PKG_CHECK_MODULES(XML2, libxml-2.0)
-if test "x$with_zypp" = "xyes"; then
-   PKG_CHECK_MODULES(JSONC, json-c, [], [AC_MSG_WARN([Cannot find json-c. Please install libjson-c-devel])])
-fi
+PKG_CHECK_MODULES(JSONC, json-c, [], [AC_MSG_WARN([Cannot find json-c. Please install libjson-c-devel])])
 
 AC_CHECK_HEADER(acl/libacl.h,[],[AC_MSG_ERROR([Cannout find libacl headers. Please install libacl-devel])])
 
