--- snapper-0.8.15/data/Makefile.am~	2021-03-15 21:35:19.000000000 +0100
+++ snapper-0.8.15/data/Makefile.am	2021-03-15 21:38:52.026915450 +0100
@@ -22,13 +22,13 @@
 	install -D -m 644 org.opensuse.Snapper.service $(DESTDIR)/usr/share/dbus-1/system-services/org.opensuse.Snapper.service
 
 if ENABLE_SYSTEMD
-	install -D -m 644 timeline.service $(DESTDIR)/usr/lib/systemd/system/snapper-timeline.service
-	install -D -m 644 timeline.timer $(DESTDIR)/usr/lib/systemd/system/snapper-timeline.timer
-	install -D -m 644 cleanup.service $(DESTDIR)/usr/lib/systemd/system/snapper-cleanup.service
-	install -D -m 644 cleanup.timer $(DESTDIR)/usr/lib/systemd/system/snapper-cleanup.timer
-	install -D -m 644 boot.service $(DESTDIR)/usr/lib/systemd/system/snapper-boot.service
-	install -D -m 644 boot.timer $(DESTDIR)/usr/lib/systemd/system/snapper-boot.timer
-	install -D -m 644 snapperd.service $(DESTDIR)/usr/lib/systemd/system/snapperd.service
+	install -D -m 644 timeline.service $(DESTDIR)/lib/systemd/system/snapper-timeline.service
+	install -D -m 644 timeline.timer $(DESTDIR)/lib/systemd/system/snapper-timeline.timer
+	install -D -m 644 cleanup.service $(DESTDIR)/lib/systemd/system/snapper-cleanup.service
+	install -D -m 644 cleanup.timer $(DESTDIR)/lib/systemd/system/snapper-cleanup.timer
+	install -D -m 644 boot.service $(DESTDIR)/lib/systemd/system/snapper-boot.service
+	install -D -m 644 boot.timer $(DESTDIR)/lib/systemd/system/snapper-boot.timer
+	install -D -m 644 snapperd.service $(DESTDIR)/lib/systemd/system/snapperd.service
 endif
 
 if HAVE_ZYPP
